core/server/data/migrations/versions/3.22/04-populate-settings-groups-and-flags.js

```diff
@@ -0,0 +1,104 @@
+const logging = require('../../../../../shared/logging');
+
+const typeGroupMapping = [{
+    keys: [
+        'members_public_key',
+        'members_private_key',
+        'members_email_auth_secret'
+    ],
+    from: 'members',
+    to: 'core'
+}, {
+    keys: [
+        'members_private_key',
+        'members_email_auth_secret',
+        'title',
+        'description',
+        'logo',
+        'cover_image',
+        'icon',
+        'accent_color',
+        'lang',
+        'timezone',
+        'codeinjection_head',
+        'codeinjection_foot',
+        'facebook',
+        'twitter',
+        'navigation',
+        'secondary_navigation',
+        'meta_title',
+        'meta_description',
+        'og_image',
+        'og_title',
+        'og_description',
+        'twitter_image',
+        'twitter_title',
+        'twitter_description'
+
+    ],
+    from: 'blog',
+    to: 'site'
+}, {
+    keys: ['amp'],
+    from: 'blog',
+    to: 'amp'
+}, {
+    keys: ['laps'],
+    from: 'blog',
+    to: 'laps'
+}, {
+    keys: ['slack'],
+    from: 'blog',
+    to: 'slack'
+}, {
+    keys: ['unsplash'],
+    from: 'blog',
+    to: 'unsplash'
+}, {
+    keys: ['shared_views'],
+    from: 'blog',
+    to: 'views'
+}, {
+    keys: ['bulk_email_settings'],
+    from: 'bulk_email',
+    to: 'email'
+}];
+
```

Comment:
@kevinansfield Adding new group column in previous migration adds the default value of `group` as `core` for all settings, and since the mapping here does not define the group values that don't change - `members`, `private`, `portal`, this leaves `group` value for them as `core` in DB after the migrations.

Think we'll need to add default mapping for these groups as well even if they are not changing in terms of value.

Reply:
@rishabhgrg I'm not quite sure what you mean here. In this migration we explicitly set the `group` value for each key https://github.com/TryGhost/Ghost/pull/11954/files#diff-33ccbf899a1b5a81c02f4043d7c48b1dR80

The only settings left with a group of `core` are the ones that are meant to be `core`

Reply:
Hangon, I see what you mean, I missed some groups in the map 👍 Fix incoming

Reply:
@rishabhgrg I've added a new map of keys and their groups where the type/group is the same - hopefully that covers everything now 😄 Thanks for the 🎣!